index.js

```diff
@@ -199,13 +199,13 @@ var Base = Class.extend({
     }
 
     var pkSql = '';
-    if (primaryKeyColumns.length > 1) {
-      pkSql = this._handleMultiPrimaryKeys(primaryKeyColumns);
-
-    } else {
-
-      primaryKeyColumns[0] = primaryKeyColumns[0].name;
-      columnDefOptions.emitPrimaryKey = true;
+    if (primaryKeyColumns.length) {
```

Comment:
@wzrdtales I'm just re-reading this now. Do you mean in the second `if` to change the `else` into an `else if`?